tests/test_api/test_types/test_telegram_object.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from aiogram import Bot
+from aiogram.api.types import TelegramObject
+
+
+class TestTelegramObject:
+    @pytest.fixture()
```

Comment:
if you want to return a bot instance and then reset it, shouldn't you use [@pytest.yield_fixture](https://docs.pytest.org/en/2.7.3/yieldfixture.html) decorator?

Reply:
`yield_fixture()` is deprecated in this version of pytest and it is recommended to use just `@pytest.fixture()`.

https://docs.pytest.org/en/latest/yieldfixture.html

Reply:
got it. thank you!